Src/WitsmlExplorer.Api/Jobs/CopyLogDataJob.cs

```diff
@@ -1,17 +1,10 @@
-using System.Collections.Generic;
 using WitsmlExplorer.Api.Jobs.Common;
 
 namespace WitsmlExplorer.Api.Jobs
 {
     public class CopyLogDataJob
     {
-        public LogCurvesReference LogCurvesReference { get; set; }
-        public LogReference Target { get; set; }
-    }
-
-    public class LogCurvesReference
-    {
-        public LogReference LogReference { get; set; }
-        public IEnumerable<string> Mnemonics { get; set; } = new List<string>();
+        public LogCurvesReference SourceLogCurvesReference { get; set; }
```

Comment:
Nitpicking, but when pluralizing in the middle of a name it is easy to overlook, personally I prefer to have it at the end e.g. `SourceLogCurveReferences` and `LogCurveReferences`